HTML/Git.html

```diff
@@ -0,0 +1,98 @@
+            <a href="HTML.html" style="color:white; text-decoration: none; font-size:20px; ">HTML</a>
+        </li>
+        <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+            <a href="Javascript.html" style="color:white; text-decoration: none;">Javascript</a>
+        </li>
+        <li style="display: block; float: left; margin-right:27px; font-size:20px;">
+            <a href="Ghibli.html" style="color:white; text-decoration: none;">Studio Ghibli</a>
+        </li>
+    </ul>
+<h2>
+    Git
+</h2>
+<div>
+    Git ist ein beliebtes VCS und wurde 2005 von Linus Torvalds, dem Entwickler hinter Linux, erstellt.<br>
+    Es konnte sich aufgrund seiner hohen Sicherheit und Effizienz durchsetzen und Verteilte, gleichzeitig ablaufende Arbeitsvorgänge.
+    Die Versionskontrolle geschieht hier durch das Branch System. Will ein<br> Entwickler z.B. änderungen an einem Code machen kann dieser eine Kopie von der neusten Version machen und dann dieses in einen unabhängigen Branch bearbeiten und testen.
+    <br>Diese änderungen werden zuallererst im Lokalen Repository gespeichert und werden dann dem Team per pull request zur Verfügung gestellt.<br>
+    Git kann und wird dabei häufig zusammen mit GitHub verwendet. Es dient hierbei als remote repository und erleichtert entwicklern die Zusammenarbeit.</div> <img src="https://de.wizcase.com/wp-content/uploads/2022/03/GitHub-Logo.png" width="200px;"><br>
+<div>Das Programm selber wird mithilfe von den Git kommandos kontrolliert:</div>
+
+
+<table style="border-style:solid">
+<tr style="border-style:solid">
+
+    <th>Befehl</th>
+    <th>Funktion</th>
+
+</tr>
+<tr style="border-style:solid; padding:8px">
+    <td style="background-color:#dddddd;padding-left:10px">
+       Git add
+    </td>
+    <td style="background-color:#dddddd;padding-left:10px">
+        Fügt änderung zum Stage hinzu, werden im nächsten Commit aufgenommen
+    </td>
+    <td> </td>
+
+</tr>
+    <tr>
+        <td style="padding-left:8px; padding-right:8px;">
+            Git commit
+        </td>
+        <td>
+            Fügt Änderung ins lokale Repository hinzu
+        </td>
+
+    </tr>
+    <tr>
+        <td style="background-color:#dddddd;padding-left:10px">
+            Git push
+        </td>
+        <td style="background-color:#dddddd;padding-left:10px">
+            Pusht branch ins Remote repository, kann jetzt von anderen Entwicklern im Team eingesehen werden
+        </td>
+
+    </tr>
+    <tr>
+        <td style="padding-left:8px;">
+            Git merge
+        </td>
+        <td>
+            Integriert Branch in master branch
+        </td>
+
+    </tr>
+    <tr>
+        <td>Workflow</td>
+        <td><a href="https://www.earthdatascience.org/images/earth-analytics/git-version-control/git-add-commit.png"> <img src="https://www.earthdatascience.org/images/earth-analytics/git-version-control/git-add-commit.png" width="200px;"></a></td>
```

Comment:
:hammer_and_wrench: Besser lesbar hiermit
```suggestion
        <td>
            <a href="https://www.earthdatascience.org/images/earth-analytics/git-version-control/git-add-commit.png">
                <img src="https://www.earthdatascience.org/images/earth-analytics/git-version-control/git-add-commit.png"
                     width="200px;">
            </a>
        </td>
```